### PR TITLE
fix svgo export example

### LINKED ISSUE #169: Context filled with empty objects
## Expected Behavior

Given I follow the documentation for the example SVGO Export
And I create a new project in Sketch using File > New From Template > iOS App Icon. 
When I select an Icon and Export Slice as SVG
Then SVG Should be optimized

## Actual Behavior

Given I follow the documentation for the example SVGO Export
And I create a new project in Sketch using File > New From Template > iOS App Icon. 
When I select an Icon and Export Slice as SVG
Then SVG has not been optimized by SVGO

## Steps to Reproduce the Problem

  1. In your home directory, [Follow the SVGO Export Documentation](https://github.com/BohemianCoding/SketchAPI/blob/develop/examples/svgo-export/README.md)
  2. Open Sketch App
  3. Create a new project in Sketch using File > New From Template > iOS App Icon. 
  4. Select bottom left Fox Icon
  5. Change export type from PNG to SVG in bottom right of right panel on window
  6. Select Export in top right of window
  7. Export all slices to home Directory
  8. Inspect exported SVG File

## Specifications

  - Version: OSX 10.13.4
  - Platform: OSX
  - Subsystem: Sketch App v49.3 (51167)

## Observation

It appears that when logging the context object, it is full of empty objects as shown below.

https://github.com/BohemianCoding/SketchAPI/blob/a1c764c67523b5e44877a0bab4e26c9ec8f91abe/examples/svgo-export/src/svgo-export.js#L79

```
export function onExportSlices(context) {
    log(`context: ${JSON.stringify(context)}`)
```

```json
context: {"plugin":{},"action":{},"scriptURL":{},"command":{},"scriptPath":{},"actionContext":{"exports":{},"document":{}}}
```

Consequentially, this causes 

```
const pathToSVGO = context.plugin.urlForResourceNamed(
    'node_modules/.bin/svgo'
  )
```

to evaluate to `null` which means that the bash command for svgo never optimizes the SVG files in the target directory.

### PR DESCRIPTION
**Pull request by mathieudutour**
Fix #169

### CODE REVIEW

**Comment by robintindale** [comment]
Hmm I'm getting
```
error Error while building ./svgo-export.js
./src/svgo-export.js
Module build failed: Error: [BABEL] /Users/Robin/git/github/SketchAPI/examples/svgo-export/src/svgo-export.js: .env is not allowed in preset options
    at /Users/Robin/git/github/SketchAPI/examples/svgo-export/node_modules/@babel/core/lib/config/validation/options.js:66:13
    at Array.forEach (<anonymous>)
    at validate (/Users/Robin/git/github/SketchAPI/examples/svgo-export/node_modules/@babel/core/lib/config/validation/options.js:64:21)
    at /Users/Robin/git/github/SketchAPI/examples/svgo-export/node_modules/@babel/core/lib/config/full.js:251:36
    at cachedFunction (/Users/Robin/git/github/SketchAPI/examples/svgo-export/node_modules/@babel/core/lib/config/caching.js:42:17)
    at loadPresetDescriptor (/Users/Robin/git/github/SketchAPI/examples/svgo-export/node_modules/@babel/core/lib/config/full.js:243:45)
    at /Users/Robin/git/github/SketchAPI/examples/svgo-export/node_modules/@babel/core/lib/config/full.js:68:19
    at Array.map (<anonymous>)
    at recurseDescriptors (/Users/Robin/git/github/SketchAPI/examples/svgo-export/node_modules/@babel/core/lib/config/full.js:66:36)
    at recurseDescriptors (/Users/Robin/git/github/SketchAPI/examples/svgo-export/node_modules/@babel/core/lib/config/full.js:97:26)
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! svgo_export@0.1.0 build: `skpm-build`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the svgo_export@0.1.0 build script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
```
this is with a clean `npm install`

**Comment by mathieudutour** [comment]
should be fixed now

File: examples/svgo-export/src/svgo-export.js [hunk 1]
```diff
@@ -39,12 +42,12 @@ export function onExportSlices(context) {
 
     // Now we run through each one...
     paths.forEach(path => {
-      log(`Compressing SVG files in ${path}`)
+      console.log(`Compressing SVG files in ${path}`)
```
Comment (robintindale) [inline examples/svgo-export/src/svgo-export.js]: I don't think this works in Sketch < 50? Is https://github.com/skpm/console required?

Comment (robintindale) [reply examples/svgo-export/src/svgo-export.js]: it actually might be nice to switch these to `sketch/ui` anyway :)

File: examples/svgo-export/src/svgo-export.js [hunk 2]
```diff
@@ -58,38 +61,26 @@ export function onExportSlices(context) {
...
 // The SVGO options are based on our experience working with Sketch's exported SVGs, and to the best of our knowledge
 // they shouldn't effect the rendering of your assets, just reduce their size.
 function optimizeFolderWithSVGO(context, folderPath) {
-  const pathToSVGO = context.plugin.urlForResourceNamed(
-    'node_modules/.bin/svgo'
+  const pathToSVGO = context.plugin
+    .urlForResourceNamed('node_modules/svgo/bin/svgo')
+    .path()
+  return execSync(
+    `${pathToSVGO} --folder='${folderPath}' --pretty --disable=convertShapeToPath --enable=removeTitle --enable=removeDesc --enable=removeDoctype --enable=removeEmptyAttrs --enable=removeUnknownsAndDefaults --enable=removeUnusedNS --enable=removeEditorsNSData`
```
Comment (robintindale) [inline examples/svgo-export/src/svgo-export.js]: btw I found compressing the whole folder a bit weird - I only want it to compress what I've exported as I might actually not want it to touch certain svg

**Comment by robintindale** [comment]
oh and yeah the copying is much faster - maybe due to webpack 4? It's like 4s instead of 10s - still not 1s of using the copy plugin but I take your point that it adds complexity to a user new to bundling

File: examples/svgo-export/src/svgo-export.js (hunk 2 above)
Comment (mathieudutour) [reply examples/svgo-export/src/svgo-export.js]: you are right. let's do that in another PR if it doesn't add complexity.

Comment (robintindale) [reply examples/svgo-export/src/svgo-export.js]: sure, will patch

File: examples/svgo-export/src/svgo-export.js (hunk 1 above)
Comment (robintindale) [reply examples/svgo-export/src/svgo-export.js]: heh ok, polyfilling might be nice rather than confusing people on Sketch < 50

**Pull request merged**